ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue

```diff
@@ -161,6 +171,10 @@ export default defineComponent({
       })
     }
 
+    const handleDialogResp = () => {
+      localState.showCancelDialog = false
+    }
```

Comment:
Simple enough that you could do this in the template on the event. 

ie line 9 ` @proceed="showCancelDialog = false"`

Reply:
Makes sense..  change made